# Program for module "Viromics Bioinformatics 2024"

This module consists of 60 working hours to be held in two weeks. Attendance is required between 9h to 12h and 13h to 16h. If not otherwise indicated by the teacher, theory will be covered in the mornings and practice in the afternoons.   

The theoretical part will be composed of a series of questions for you to research and answer from 9h to 11h30. From 11h30 to 12h you will present and discuss your findings to a teacher or teacher assistant (TA). In the afternoon, you will run tools and analyse data. From 15h30 to 16h you will discuss your advances with a teacher or TA. At any point in time, they will be available to answer questions.       

- Taxonomy I: Cluster proteins, Vcontact network, classification of taxonomy based on clusters
- Taxonomy II: Genomad, HMM search for marker genes

**Week 1:**

- **MON 16/Sep/2024: Introduction**
  - 9h - 10h:
    - Introduction to the program and evaluation criteria: [lab book](https://github.com/waltercostamb/course_viromics-bioinformatics_2024/blob/main/tutorials/1.0_lab_book.md) and presentation 
    - Students introduce themselves and reflect on the program
    - Access to high performance cluster draco

  - 10h - 11h: [1.1_introduction_viromics](https://github.com/waltercostamb/course_viromics-bioinformatics_2024/blob/main/tutorials/1.1_introduction_viromics.md)
  - 12h - 13h: lunch time
  - 13h - 16h: [1.2_sequencing_quality](https://github.com/waltercostamb/course_viromics-bioinformatics_2024/blob/main/tutorials/1.2_sequencing_quality.md)

- **TUE 17/Sep/2024: Assembly**
  - 9h - 12: [Assembly I](https://github.com/waltercostamb/course_viromics-bioinformatics_2024/blob/main/tutorials/2.0_assembly_I.md)
  - 12h - 13h: lunch time
  - 13h - 16h: [Assembly II](https://github.com/waltercostamb/course_viromics-bioinformatics_2024/blob/main/tutorials/2.1_assembly_II.md)
 
- **WED 18/Sep/2024: Identifying viruses**
  - 9h - 12: Read benchmark paper: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03236-4
    - focus on understanding Figures 1, 3 and 6
  - 12h - 13h: lunch time
  - 13h - 16h: Identify viral contigs, assess genome completeness, filter only high quality ones

- **THU 19/Sep/2024: Gene finding**
  - 9h - 12: talk/research annotation
  - 12h - 13h: lunch time
  - 13h - 16h: find genes
 
- **FRI 20/Sep/2024: Functional annotation and visualization**
  - 9h - 12: with HMMsearch, search for PHROGs (if it takes too long, select longest contigs)
  - 12h - 13h: lunch time
  - 13h - 16h: visualization (if it takes too long, select longest contigs)
    - ggplot or a python package (show annotations in a single contig plot)
    - easyfig (more than one contig): https://mjsull.github.io/Easyfig/ - depends on clustering method (AAI?)
 
**Week 2:**
 
- **MON 23/Sep/2024: Host annotation**
  - 9h - 12:
  - 12h - 13h: lunch time
  - 13h - 16h

- **TUE 24/Sep/2024: Taxonomy and Phylogeny**
  - 9h - 12: Viral taxonomy vs. microbial taxonomy
  - 12h - 13h: lunch time
  - 13h - 16h: Genomad and VContact2

- **WED 25/Sep/2024: Research question**
  - 9h - 12: 2.3.x_research_question.md
  - 12h - 13h: lunch time
  - 13h - 16h: Develop your reseach proposal
 
- **THU 26/Sep/2024: Prepare for presentation**
  - 9h - 12: Open hour for questions, further develop your reseach proposal
  - 12h - 13h: lunch time
  - 13h - 16h: prepare for presentation

- **FRI 27/Sep/2024: Presentation**
  - 10h - 11h30: Final presentation: 20' presentation plus questions
  - ?
  - 11h30 - 13h: lunch time
  - 13h - 16h: Clean up documentation and scripts and send them to the teachers
