# Program for module "Viromics Bioinformatics 2024"

This module consists of 60 working hours to be held in two weeks. Attendance is required between 9h to 12h and 13h to 16h. If not otherwise indicated by the teacher, theory will be covered in the mornings and practice in the afternoons.   

The theoretical part will be composed of a series of questions for you to research and answer from 9h to 11h30. From 11h30 to 12h you will present and discuss your findings to a teacher or teacher assistant (TA). In the afternoon, you will run tools and analyse data. From 15h30 to 16h you will discuss your advances with a teacher or TA. At any point in time, they will be available to answer questions.      

**Week 1:**

- **MON 16/Sep/2024: Introduction**
  - 9h - 10h:
    - Introduction to the program
    - Introduction of the students
    - Accession to high performance cluster draco
    - Discussion and decision of entry point into program (what topics do the students already know?)
    - Evaluation criteria: [lab book](https://github.com/waltercostamb/course_viromics-bioinformatics_2024/blob/main/tutorials/1.0_documentation.md) and presentation 
  - 10h - 11h: [1.1_introduction_viromics](https://github.com/waltercostamb/course_viromics-bioinformatics_2024/blob/main/tutorials/1.1_introduction_viromics.md)
  - 12h - 13h: lunch time
  - 13h - 16h: [1.2_sequencing_quality](https://github.com/waltercostamb/course_viromics-bioinformatics_2024/blob/main/tutorials/1.2_sequencing_quality.md)

- **TUE 17/Sep/2024: Assembly**
  - 9h - 12: [Assembly I](https://github.com/waltercostamb/course_viromics-bioinformatics_2024/blob/main/tutorials/2.0_assembly_I.md)
  - 12h - 13h: lunch time
  - 13h - 16h: [Assembly II](https://github.com/waltercostamb/course_viromics-bioinformatics_2024/blob/main/tutorials/2.1_assembly_II.md)
 
- **WED 18/Sep/2024: Identifying and annotating viruses I**
  - 9h - 12: Read benchmark paper: https://www.biorxiv.org/content/10.1101/2023.04.26.538077v2.full
  - 12h - 13h: lunch time
  - 13h - 16h: Jaeger, Virify and a 3rd tool on a virome and a metagenome

- **THU 19/Sep/2024: Identifying and annotating viruses II**
  - 9h - 12: talk/research annotation
  - 12h - 13h: lunch time
  - 13h - 16h: analyse and interprete results from yesterday
 
- **FRI 20/Sep/2024: Taxonomy**
  - 9h - 12: Viral taxonomy vs. microbial taxonomy
  - 12h - 13h: lunch time
  - 13h - 16h: VContact2
 
**Week 2:**
 
- **MON 23/Sep/2024: Data analysis: answering a research question**
  - 9h - 12: 
  - 12h - 13h: lunch time
  - 13h - 16h: 

- **TUE 24/Sep/2024: Data visualization I**
  - 9h - 12:
  - 12h - 13h: lunch time
  - 13h - 16h

- **WED 25/Sep/2024: Data visualization II**
  - 9h - 12:
  - 12h - 13h: lunch time
  - 13h - 16h
 
- **THU 26/Sep/2024: Prepare for presentation**
  - 9h - 12: Open hour for questions
  - 12h - 13h: lunch time
  - 13h - 16h:

- **FRI 27/Sep/2024: Presentation**
  - 9h - 12: Clean up documentation and scripts and send them to the teachers
  - 12h - 13h: lunch time
  - 13h - 16h: Final presentation: 20' presentation plus questions
 

- Source: https://mgxlab.github.io/Viromics-Workshop-MGX/
- Taxonomy I: Cluster proteins, Vcontact network, classification of taxonomy based on clusters
- Taxonomy II: Genomad, HMM search for marker genes
