# Assembly II

## Theory

We will use SPAdes to assemble our viromes.  "A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing", by Bankevich et al 2012. Link: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3342519/


Answer the following questions:

----

- What is computer programming?
- Cite more or less 5 characteristics of each of the following languages:
  - bash
  - python
  - R
  - C
  - java

-----

Afterwards, search the topics:

----

- What is GITHub?
- Why is it useful for computational biology?

----

Do the bash tutorial of the following page: https://www.freecodecamp.org/news/bash-scripting-tutorial-linux-shell-script-and-command-line-for-beginners/. Skip "How to Schedule Scripts using cron" and "How to Debug and Troubleshoot Bash Scripts"

## Practice

Check how many lines you have in your assembly output. Check the file head and tail. Count how many contigs/scaffolds you have using bash scripts.  

Evaluate the assembly you did yesterday with checkV: https://bitbucket.org/berkeleylab/checkv/src/master/. First, understand what this tool does and write a command line to run it.   

Follow the tutorial of "Sequence Quality" chapter of Dr Aldert Zomer's course: https://aldertzomer.github.io/Microbial-Genomics-2022/50_Assembly_Quality/index.html. Adapt the tutorial to use checkv instead of checkm.

Finally, answer the following question:

----

Is the assemble complete or fragmented?

----
